src/d3a/d3a_core/export.py

```diff
@@ -517,6 +607,26 @@ def plot_bar_graph(cls, barmode: str, title: str, xtitle: str, ytitle: str, data
         fig = go.Figure(data=data, layout=layout)
         py.offline.plot(fig, filename=iname, auto_open=False)
 
+    @classmethod
+    def plot_line_graph(cls, title: str, xtitle: str, ytitle: str, data, iname: str, xmax: int):
+        layout = go.Layout(
```

Comment:
Isnt that the point where you can increase the line thickness? Or did I miss it somewhere else?

Reply:
https://github.com/gridsingularity/d3a/pull/366/files#diff-9e95bed0f62db1b1f967c65f0823d49dR418

Reply:
As this is just a function for the layout, please rename.